packages/quantic/force-app/main/default/lwc/descriptionStrengthIndicator/descriptionStrengthIndicator.css

```diff
@@ -0,0 +1,3 @@
+.h3_heading{
+    font-weight: 600;
```

Comment:
Is there really a variation of 600 font-weight?
Usually it's 400 for normal, or 700 for bold I think?

Reply:
Yes, It's really hard to see the difference between the 600 and 700 font weights but it exists x)

Reply:
Is this from the mockups? sometimes it's not 100% intentional on the designer's part and if 700 would do then we could eliminate this file completely. Might be worth checking.

Reply:
Yess ! I will remove that and get the bold title using the slds class : `slds-text-title_bold`